SPIFlashLogger.class.nut

```diff
@@ -275,8 +472,19 @@ class SPIFlashLogger {
         }
     }
 
+    //
+    // Increase sector id and check id overflow
+    //
+    function _getNextSectorId() {
+        if (_nextSectorId == 0x7FFFFFFF || _nextSectorId == 0)
```

Comment:
Isn't it the same as 
```
if (_nextSectorId < 0) {
   _nextSectorId = 1;
}
_nextSectorId++;
```
Also I feel like there is a bug here. If _nextSectorId == 0 it turns into 2, while I feel like it should probably be 1. Shouldn't it?

Do we have a test case that covers these corner cases? I didn't notice any.

Reply:
Please, pay attention that code has a return like:
return nextSectorId++;

So, if nextSectorId == 1 then it will return 1

it is easier to replace it with 
```
if (_nextSectorId =< 0)
   _nextSectorId = 1;
```

Reply:
_nextSectorId === 0 could happen on init only, and it means that all sectors are clean